tensorflow_datasets/core/dataset_builder.py

```diff
@@ -28,11 +28,16 @@
 
 from absl import logging
 import six
-import tensorflow.compat.v2 as tf
+import tensorflow.compat.v2 as tf  # type: ignore
+import apache_beam as beam
 
+# pytype: disable=pyi-error
+from tensorflow_datasets import ReadConfig
```

Comment:
Please do not use direct object import. Use `read_config_lib.ReadConfig` instead

Reply:
Done